src/sage/interfaces/magma.py

```diff
@@ -1303,33 +1303,37 @@ def _object_class(self):
     # It's very painful using the interface without this.
     def _left_list_delim(self):
         """
-        Return the left sequence delimiter in Magma. Despite the name in
+        Return the left sequence delimiter in Magma.
+
+        Despite the name in
         this function, this is really the least painful choice.
 
         EXAMPLES::
 
             sage: magma._left_list_delim()
             '['
         """
-        #return "[*"
+        # return "[*"
         return "["
```

Comment:
I think these commented code with no explanation why they are there should be eliminated.